app/survey_config/business_config.py

```diff
@@ -55,10 +56,21 @@ def __post_init__(self):
         ]
 
     def get_service_links(
-        self, sign_out_url: str, *, is_authenticated: bool
+        self, sign_out_url: str, *, is_authenticated: bool, ru_ref: Optional[str]
     ) -> Optional[list[dict]]:
+        if self.schema and is_authenticated:
```

Comment:
We should check that we have `ru_ref` also.

Reply:
Check added, thanks!